libraries/Microsoft.Bot.Builder.AI.LUIS/LuisExtensions.cs

```diff
@@ -29,7 +32,7 @@ public static IConfigurationBuilder UseLuisSettings(this IConfigurationBuilder b
             var luisRegion = configuration.GetValue<string>("LUIS_AUTHORING_REGION") ?? configuration.GetValue<string>("region") ?? "westus";
             var environment = configuration.GetValue<string>("environment") ?? Environment.UserName;
             var settings = new Dictionary<string, string>();
-            settings["luis:endpoint"] = $"https://{luisRegion}.api.cognitive.microsoft.com";
+            settings["luis:endpoint"] = configuration.GetValue<string>("luis:endpoint") ?? $"https://{luisRegion}.api.cognitive.microsoft.com";
```

Comment:
How is this related to the issue associated with this PR?

Reply:
I have discarded this change.